executor/executor_test.go

```diff
@@ -6481,3 +6481,46 @@ from ssci right join csci on (ssci.customer_sk=csci.customer_sk
                                and ssci.item_sk = csci.item_sk)
 limit 100;`)
 }
+
+func TestIssue42662(t *testing.T) {
```

Comment:
Because too many test cases are in the executor. Please move it into ```executor/issuetest```. 

on the other hand, ```TestIssuexxxx``` makes people wonder what exactly has been repaired.